pom.xml

```diff
@@ -16,7 +16,7 @@
 
   <name>Jenkins Git client plugin</name>
   <description>Utility plugin for Git support in Jenkins</description>
-  <url>https://wiki.jenkins.io/display/JENKINS/Git+Client+Plugin</url>
+  <url>https://plugins.jenkins.io/git-client</url>
```

Comment:
You want to link to the wiki; otherwise there will be a redirect loop on `plugins.jenkins.io` and there will be no way to find the wiki. By “canonical URL” I meant the `jenkins.io` version as opposed to `jenkins-ci.org`.